test/widgets/inbox_test.dart

```diff
@@ -252,6 +265,28 @@ void main() {
                 || widget.icon == ZulipIcons.arrow_right))));
       }
 
+      Future<void> dragUntilInvisible(
```

Comment:
Let's pull this out to the library toplevel, above `main`. It's a generic test helper that we might want to move to a more central file in order to reuse, and that would also make sense to go upstream; so that helps highlight it for such reuse.

Reply:
For similar reasons, let's give this method a doc comment. It can be largely cribbed from the one on `dragUntilVisible` upstream.